client/src/App.tsx

```diff
@@ -210,6 +216,90 @@ function App() {
 		}
 	}, []);
 
+	const activateFireworks = () => {
+		var duration = 2 * 1000;
```

Comment:
don't ever use var, either use const or let
in this case duration should be const, and ideally defined outside the function since it's a static value, something like fireworkDuration